Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -660,6 +660,7 @@ private extension SPNoteListViewController {
             UIContextualAction(style: .destructive, image: .image(name: .trash), backgroundColor: .simplenoteDestructiveActionColor) { (_, _, completion) in
                 SPTracker.trackListNoteDeleted()
                 SPObjectManager.shared().permenentlyDeleteNote(note)
+                self.setEmptyTrashEnabled()
```

Comment:
A better sport for this one would be `startDisplayingEntities`. If you empty the `trash` in a 2nd device / web, the Trash button would remain enabled otherwise.